SPECS/sm.spec

```diff
@@ -419,6 +420,10 @@ cp -r htmlcov %{buildroot}/htmlcov
 %{_unitdir}/linstor-monitor.service
 
 %changelog
+* Thu Apr 29 2021 Ronan Abhamon <ronan.abhamon@vates.fr> - 2.30.3-1.2
```

Comment:
To be merged with the previous changelog entry, and remove the WIP and TODO lines from it. You can put your name instead of mine since you did most of the work :)